src/main/java/com/softserve/academy/spaced/repetition/service/DeckCommentService.java

```diff
@@ -3,18 +3,51 @@
 import com.softserve.academy.spaced.repetition.domain.Comment;
 import com.softserve.academy.spaced.repetition.domain.DeckComment;
 import com.softserve.academy.spaced.repetition.utils.exceptions.NotAuthorisedUserException;
+
 import java.util.List;
 
+/**
+ * This interface works with the comments to decks.
+ */
 public interface DeckCommentService {
-
+    /**
+     * Add a comment to decks
+     *
+     * @return true if operation is successful
+     */
     DeckComment addCommentForDeck(Long deckId, String commentText, Long parentCommentId)
             throws NotAuthorisedUserException;
 
+    /**
+     * Returns a comment with the given identifier
+     *
+     * @param commentId must not be {@literal null}.
```

Comment:
as for me, if meaning of parameter is obvious, would be better to have `@param commentId comment id`, but try to find the same example in oracle code

Reply:
corrected